src/pint/models/binary_ell1.py

```diff
@@ -267,3 +269,85 @@ def validate(self):
         super().validate()
         # if self.H3.quantity is None:
         #     raise MissingParameter("ELL1H", "H3", "'H3' is required for ELL1H model")
+
+
+class BinaryELL1k(BinaryELL1):
```

Comment:
Should this give the primary full reference info of Susobhanan et al? (with an ADS link)?

Reply:
The citation conventions in PINT source code not very consistent. For instance, long-form citation is not given for Lange et al (ELL1), but is given for Freire & Wex (ELL1H). Making this consistent can be a separate PR I think.

Reply:
As I have been going over various binary models I have been updating it (see e.g., the DDK model, or #1490 ). But I guess it's fine to leave.

Reply:
OK... I have changed the references in ELL1, ELL1H and ELL1k to look similar to how it is done in DDK.